ci/pipelines/apps-test-ecr.yml

```diff
@@ -25,6 +25,13 @@ resources:
       branch: master
       paths:
         - ci/pipelines/apps-test-ecr.yml
+  - name: pay-ci
```

Comment:
Might be for another PR but we could probably get rid of the `apps-test-ecr` resource now there is a `pay-ci` one.

Reply:
yep, I'll tidy this up in another PR